### DISCUSSION TITLE
Scheduled assets with cron not working as expected

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17837] [Q&A]
Hey guys , anyone know why the scheduling assets doesn’t work with cron schedule. There is no materialization at the specified time .

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699475832997439?thread_ts=1699475832.997439&cid=C01U954MEER).*

**Comment by jamiedemaria** [comment]
Hi, can you share some code of your schedule and the assets you're trying to materialize? Any log messages you get?
